spec/stack.js

```diff
@@ -18,19 +18,52 @@ describe('Stack', () => {
       expect(() => aStack.push('foo'))
         .to.alter(() => aStack.length(), { from: 0, to: 1 })
     })
+    it('pushes an element to the top of the stack.', () => {
+      const aStack = new Stack()
+
+      aStack.push('bar')
+      aStack.push('foo')
+
+      expect(() => aStack[0]
+        .to.equal('foo'))
```

Comment:
I don't think lines 27 & 28 are doing what they're supposed to be doing.
The right syntax is `expect( aStack[0] ).to.equal('foo')`.  But this is also not good because `aStack[0]` is not a thing.  You can't access a stack the same way as an array.  If you want to look at the element on the top of the stack you must use `peek()` or `pop()`.